mysqld_exporter.go

```diff
@@ -78,6 +82,7 @@ const (
 	informationSchema = "info_schema"
 	performanceSchema = "perf_schema"
 	slaveStatus       = "slave_status"
+	binlog            = "binlog"
```

Comment:
I'm trying to decide if I like this name.  What do you think of calling this `master_logs` to match the source of the data.  This way the final metric name would be `mysql_master_logs_size_total`.